doc/framework-integration/angular-instructions.md

```diff
@@ -39,33 +39,60 @@ to work properly.
 To use the inovex elements, you need to integrate the `InoElementsModule` into one of your App's
 modules. The main module is usually called `AppModule`. If you have more than one module, you should
 integrate the `InoElementsModule` into each module that is supposed to use the inovex elements. Alternatively,
- you can also integrate the components into a shared module. This way, you only need to import the components once 
- (recommended).
+you can also integrate the components into a shared module. This way, you only need to import the components once
+(recommended).
 
-You only need two lines of code to import the package and make it available in Angular:
+We need two steps to integrate the components inside angular
+
+- Include the CUSTOM_ELEMENTS_SCHEMA in the modules that use the components.
+- Call defineCustomElements() from `main.ts` (or some other appropriate place).
+
+1. Edit `app.module.ts`
 
 ```typescript
 // src/app/app.module.ts
 
 import { BrowserModule } from '@angular/platform-browser';
-import { NgModule } from '@angular/core';
+import { CUSTOM_ELEMENTS_SCHEMA, NgModule } from '@angular/core';
 
 import { InoElementsModule } from '@inovex.de/elements-angular'; // <-- a) import our package
 
 import { AppComponent } from './app.component';
 
 @NgModule({
-  declarations: [
-    AppComponent
-  ],
+  declarations: [AppComponent],
   imports: [
     BrowserModule,
     InoElementsModule.forRoot() // <-- b) make it available in Angular
   ],
   providers: [],
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
```

Comment:
Can you point to the changes with the same type of comment we made e.g. in line 66 (`// <-- b) make it available in Angular`) like this:
```js
schemas: [CUSTOM_ELEMENTS_SCHEMA], // <-- c) <insert reason why we need this>
```